vim/vim
# Organize CI settings

### `vimtags` test

* Add and seperate the independent job to test vimtags; make the job simple.
* Make `vimtags` test target simple; have the shell itself check whether $(VIMEXE) is executable.
* Modify doctags.vim; can display `helptags` result and error message.

### Organize if_ver*.vim

* Combine into one script and move into ci/ directory
